translate/storage/csvl10n.py

```diff
@@ -25,72 +25,9 @@
 import six
 
 from translate.misc import csv_utils
-from translate.misc import sparse
```

Comment:
Just verified that `translate/misc/sparse.py` can be also dropped. Please include its removal in this PR.